### PR TITLE
Code style

### PR DESCRIPTION
**Pull request by bendavies**
Is there any reason why CS isn't enforced any more?
Consistency is good!

I've enabled caching. should work, provided that travis retains file modification times on checkout?
It also runs only under php7 to speed things up.

### CODE REVIEW

**Comment by amenophis** [comment]
:+1:

File: .gitignore [hunk 1]
```diff
@@ -24,4 +26,4 @@
 
 /behat.yml
 /phpunit.xml
-/phpspec.yml
+/phpspec.yml
```
Comment (pamil) [inline .gitignore]: To be reverted :)

**Comment by lchrusciel** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (bendavies) [inline .gitignore]: :+1:

**Comment by bendavies** [comment]
caching doesn't seem to be working for some reason.

**Comment by bendavies** [comment]
@lchrusciel sure, there are other things to skip i think, like trailing commas in arrays.

**Comment by bendavies** [comment]
@Sylius/core-team RFC now :)

**Comment by bendavies** [comment]
@lchrusciel @pamil RFC now :)

File: src/Sylius/Bundle/ArchetypeBundle/EventListener/LoadMetadataSubscriber.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ class LoadMetadataSubscriber implements EventSubscriber
     protected $subjects;
 
     /**
-     * Constructor
+     * Constructor.
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ArchetypeBundle/EventListener/LoadMetadataSubscriber.php]: We should do something with this redundant comments to.

Comment (bendavies) [inline src/Sylius/Bundle/ArchetypeBundle/EventListener/LoadMetadataSubscriber.php]: this comes from `phpdoc_short_description`. I've disabled it.

**Comment by bendavies** [comment]
any interest in my keeping this up to date?

**Comment by michalmarcinkowski** [comment]
@bendavies it would be great to have this merged, but merging all this changes will force almost every waiting PR to rebase with master and that's not something we would like to do right now...

Is there any way to run fixer just on the files that were changed in the PR?

**Comment by bendavies** [comment]
@michalmarcinkowski understandable.

There's not an easy way to do that from what I can see...

**Comment by bendavies** [comment]
@michalmarcinkowski @pjedrzejewski 
I've just gone looked through the active PR's.
At the moment, there aren't many big open PR's that touch lots of code (granted, there are a few). Most of the big PR's are code additions, not edits. 

Therefore, it *might* be a good time to merge this, as I don't think it'll be that much work for the open active PR's to rebase.

**Comment by pjedrzejewski** [comment]
@bendavies Sounds like a good plan, give me a ping when you rebased, I will merge it before others. Thanks Ben!

**Comment by bendavies** [comment]
@pjedrzejewski done. let me know if you want any rules adjusted.

**Comment by bendavies** [comment]
@pjedrzejewski do you want to use long or short array styles?

**Comment by pjedrzejewski** [comment]
@bendavies After quick poll let's go with short array syntax, I will hold all merges until you rebase. :) Thanks Ben! :+1:

**Comment by bendavies** [comment]
@pjedrzejewski done.

File: src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php [hunk 3]
```diff
@@ -342,7 +340,7 @@ private function getFormWithFields(ElementInterface $element, array $fields)
             }
         }
 
-        return null;
+        return;
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php]: Can we leave `return null`? IMHO if method returns some value, it should use `return $something` and `return null`, if method does not return anything, it should use `return`.

Comment (bendavies) [inline src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php]: of course. this is part of the symfony style. 
if you don't want this we can remove the rule.
Discuss and let me know,

Comment (bendavies) [inline src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php]: infact there is discussion here https://github.com/symfony/symfony/issues/17201

Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php]: @bendavies Yes, let's remove this rule for now and rebase pls, I will merge asap, thanks!

Comment (bendavies) [inline src/Sylius/Bundle/CoreBundle/Behat/MetadataContext.php]: done

**Comment by bendavies** [comment]
@pjedrzejewski passed

**Comment by pjedrzejewski** [comment]
Thanks Ben! :+1: Nice work!

**Comment by bendavies** [comment]
Cool! No need to review PR's for style any more!

**Comment by michalmarcinkowski** [comment]
Great! :+1:

File: src/Sylius/Behat/Page/Product/ProductShowPage.php [hunk 4]
```diff
@@ -48,7 +47,6 @@ public function addToCartWithQuantity($quantity)
     }
 
     /**
-     * @return null
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: That is strange...

Comment (bendavies) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: i'll check it out

Comment (bendavies) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: it's the `phpdoc_no_empty_return` rule.
Will fix in another pr.

Comment (michalmarcinkowski) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: :+1:

Comment (bendavies) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: https://github.com/Sylius/Sylius/pull/4019

**Pull request merged**